.github/workflows/release-binaries-setup-stage/action.yml

```diff
@@ -0,0 +1,55 @@
+name: Setup Stage
+
+inputs:
+  previous-artifact:
+    description: >-
+      A unique descriptor for the artifact from the previous stage.  This will
+      be used to construct the final artifact pattern, which is:
+      $RUNNER_OS-$RUNNER_ARCH-$PREVIOUS_ARTIFACT-*
+    required: false
+    type: 'string'
+
+outputs:
+  build-prefix:
+    description: "Directory containing the build directory."
+    value: ${{ steps.build-prefix.outputs.build-prefix }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Ninja
+      uses: llvm/actions/install-ninja@22e9f909d35b50bd1181709564bfe816eaeaae81 # main
```

Comment:
Side note: Is there a reason these actions are still in a separate repo and not in the mono-repo?

Reply:
The only advantage of having them in a separate repo is that you can use them without checking out the project source code.  However, for this specific action I don't think it would ever be used without already having the source code, so there is probably no reason to keep it here.